common/errors.go

```diff
@@ -26,3 +28,13 @@ type ServerError struct {
 func (err ServerError) Error() string {
 	return pb.XChainErrorEnum_name[int32(err.Errno)]
 }
+
+func NormalizedKVError(err error) error {
```

Comment:
is this a KV specified function? shall we put it in kv package?

Reply:
Here, NormalizedKVError is aimed to normalize errors such as leveldb and badger. In the future, we might normalize other types of errors, so I think it is a good way to put NormalizedKVError into the file of common/errors.go.

Reply:
OK